pkg/version/version.go

```diff
@@ -1,12 +1,54 @@
+// Package version gathers version data from variables set at build time and runtime
+// and provides access points to retrieve them.
 package version
 
-import "fmt"
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+)
 
 var (
 	Version   = "dev"
 	GitCommit = "HEAD"
 )
 
+const primeEnv = "RANCHER_PRIME"
```

Comment:
For my own education, is there a reason this isn't refixed with `CATTLE_`?

Reply:
The settings manager uses the `CATTLE_` prefix to look up settings in the environment, so using the prefix would imply that the variable is available as a Rancher setting (though this isn't a 100% rule). Since this isn't a setting, there's no technical reason to use the prefix. This may reach into the realm of personal preference, but `CATTLE_RANCHER_...` feels redundant to me, so I kept it short.